B827EBFFFEDAB675.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFEDAB675",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 59,33831547,
+    "ref_longitude": 18,01383567,
```

Comment:
@Nadidudida Please use period (`.`) as decimal-separator, instead of comma, otherwise it's not a valid JSON file. Let me know when you updated it, and I will merge. Thx!